### DISCUSSION TITLE
Should I install "gha-runner-scale-set-controller" chart seperately, and if so, how many?

### DISCUSSION THREAD

**Discussion by cniessigma** [actions/actions-runner-controller#2933] [FAQs] [answered]
Hello!

Perhaps I am just being a bit thick, but as a year-long user of ARC, I am looking to upgrade to using RunnerScaleSets and get some of the benefits of it. So I went to the quickstart which tells me to install the `gha-runner-scale-set` Helm chart, which is fair enough, but I intend to make many, many runner scale sets for different permutations of hardware, the same way I would now with RunnerDeployments.

I am curious if the intended design for such a thing is to install the `gha-runner-scale-set-controller` chart separately once for all the scale sets, and then install one `gha-runner-scale-set` _per_ set that I would like to manage? I can't find it documented anywhere if I should ever install the `gha-runner-scale-set-controller` directly at all

**Comment by mihirsamdarshi** [comment, marked as answer]
Yes, this is exactly the case
